frontend/app/routes/_public+/apply+/$id+/demographics-part1.tsx

```diff
@@ -75,13 +75,13 @@ export default function DemographicsPart1() {
           </div>
         )}
         <div className="flex flex-wrap items-center gap-3">
-          <ButtonLink id="cancel-button" to="/personal-information">
+          <ButtonLink id="back-button" to={`/personal-information`}>
```

Comment:
No need for JSX curly braces unless the url path is not good and requires `id`.

Is relative path working or it needs the following value `apply/${id}/personal-information`?

```suggestion
          <ButtonLink id="back-button" to="/personal-information">
```